_module_templates/macros_bash.md

```diff
@@ -0,0 +1,81 @@
+<!--
+
+author:   DART Team
+email:    dart@chop.edu
+version:  1.0.0
+current_version_description: Initial Version
+language: en
+narrator: UK English Female
+title: Bash Module Macros
+comment:  This is placeholder module to save macros used in other modules.
+
+@version_history 
+
+Previous versions: 
+
+No Previous Versions
+@end
+
+@lesson_prep_bash_basics
+
+You will get the most out of this lesson if you follow along with the examples and try out the commands. In order to do that you need to have a bash shell open on your computer. Please follow the instructions appropriate for the computer you are using.
+
+**Open a bash shell.**
+If you are using a computer with running iOS (i.e. a Mac) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.
```

Comment:
```suggestion
If you are using a computer with running a Unix operating system (i.e. a Mac or Linux computer) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.
```

I believe iOS is only for mobile devices (iphone, ipad, etc.); MacOS is the computer version. It might be clearer to just say "unix" though since that includes Linux builds.

Reply:
I'd also recommend pointing the download instructions to the specific page of the 101 module rather than to its overview

Reply:
You are so right, I copied this directly from the bash 103 module, which obviously needed some updating....